### DISCUSSION TITLE
Feature request NCINM

### DISCUSSION THREAD

**Discussion by choonsiklee** [ncidose/ncidosetools#34] [NCINM] [answered]
Please leave feature request.

**Comment by choonsiklee** [comment]
**Anonymous**

Is it possible to include other radionuclides? I am working with Sc44, Sc44m and Sc43.

**Comment by choonsiklee** [comment]
**Anonymous**

Can you please add alpha radionuclides?

**Comment by choonsiklee** [comment]
**Anonymous**

It will be great if users can add custom radionuclides to the existing list.

**Comment by choonsiklee** [comment]
**Seth W Streitmatter**

Ac-225, Cu-64 dotatate, Lu-177 dotatate

**Comment by choonsiklee** [comment]
**Anonymous**

Sc44, Sc44m and Sc43

**Comment by choonsiklee** [comment, marked as answer]
Error report: Radionuclide changes when gender is changed.

**Comment by ncidoseteam** [comment]
Batch calculation mode
